docs/source/host/quick-start.rst

```diff
@@ -0,0 +1,59 @@
+########################
+Hosting Quick Start
+########################
+
+.. contents:: On This Page
+    :depth: 2
+    
+----------------------------
+Notice: Port Forwarding
+----------------------------
+
+There are two ways to make a domain accessible from the outside internet.
```

Comment:
```suggestion
There are two ways to make a domain accessible from the outside Internet.
```

Reply:
This is a bit of an odd one, personally I wouldn't capitalize it, I don't capitalize "metaverse" either typically but apparently in America and academic forums in America they do capitalize it, so what the heck. lol 
https://en.wikipedia.org/wiki/Capitalization_of_Internet 
https://grammarist.com/style/internet-capitalization/